### PR TITLE
Remove O and 0 from generated codes

### PR DESCRIPTION
**Pull request by meganle**
Addresses #223

### CODE REVIEW

File: src/utils/AppDevUtils.js [hunk 1]
```diff
@@ -31,8 +31,15 @@ const netIDFromEmail = (email: string): string => email.substring(0, email.index
  * @param {number} length - Desired length of random code
  * @return {string} Randomly generated code
  */
-const randomCode = (length: number): string => Math.round(((36 ** (length + 1)) - Math.random()
-    * (36 ** length))).toString(36).slice(1).toUpperCase();
+const randomCode = (length: number): string => {
+  // Don't allow 0 and O to avoid confusion
+  const allowedChars = '123456789ABCDEFGHIJKLMNPQRSTUVWXYZ';
```
Comment (mindylou) [inline src/utils/AppDevUtils.js]: can we get rid of `I` too? because it looks like `1`

File: src/utils/AppDevUtils.js [hunk 2]
```diff
@@ -31,8 +31,15 @@ const netIDFromEmail = (email: string): string => email.substring(0, email.index
  * @param {number} length - Desired length of random code
  * @return {string} Randomly generated code
  */
-const randomCode = (length: number): string => Math.round(((36 ** (length + 1)) - Math.random()
-    * (36 ** length))).toString(36).slice(1).toUpperCase();
+const randomCode = (length: number): string => {
+  // Don't allow 0, O, and I to avoid confusion
+  const allowedChars = '123456789ABCDEFGHJKLMNPQRSTUVWXYZ';
+  let result = '';
+  for (let i = length; i > 0; --i) {
```
Comment (young-k) [inline src/utils/AppDevUtils.js]: why not `while (result.length < length)`? I think it would be more readable

Comment (meganle) [reply src/utils/AppDevUtils.js]: lol true i just copied from stack overflow

Comment (young-k) [reply src/utils/AppDevUtils.js]: that's a violation of academic integrity! :angry:

**Pull request merged**